### ISSUE TITLE
Difference between pretrained models

### ISSUE THREAD

**Issue by os1a** [HarshayuGirase/PECNet#19]
Hi,

I am wondering if you can give some explanation about the different pretrained models on SDD. When running the test script on these models, I got different results (the best are for the first model PECNET_social_model1.pt)

Thanks in advance,

Best,

**Comment by HarshayuGirase** [comment]
Hi,

I believe there's no major difference aside from maybe a couple of hyperparameter changes such as LR or loss function coefficients. The difference is due to variance while training but the results should be within 1-2% of each other.

**Comment by karttikeya** [comment]
Hope this answers your questions. I'm closing this issue for now. Please feel free to open another issue for other questions, star the repo if you found it helpful and thanks for interest in our code.
